mlrun/projects/pipelines.py

```diff
@@ -820,8 +820,9 @@ def run(
         namespace=None,
         source=None,
     ) -> typing.Optional[_PipelineRunStatus]:
-        workflow_name = name.split("-")[-1] if f"{project.name}-" in name else name
-
+        workflow_name = (
+            name.replace(f"{project.name}", "")[1:] if project.name in name else name
```

Comment:
what happens in this example:
```
project.name = "test"
task name = "test-workflow"
```
will the task name be `"-workflow"?

Reply:
nope, because we add the project name again to the base name ("test-test-workflow"), but i can change the name if it make sense